capabilities/identity/face-detection/profile.supr

```diff
@@ -0,0 +1,85 @@
+name = "identity/face-detection"
```

Comment:
Why did you decide to use `identity` as scope? 

I'd imagine if it was capable of verifying person's identity, but as it is I think `Computer vision` or more narrow `recognition` would be better. But just naming, so take it just as suggestion.

Reply:
I used identity because of naming in here: https://www.notion.so/superface/Capabilities-Problem-Spaces-Know-Your-Customer-4f110dd6ecfb4fe2afb773210a78da89 
Yep, pretty weak reason  😅
Computer vision is probably better- should I rename it?🤷